### PR TITLE
Feature/product section

### PR DESCRIPTION
**Pull request by elic4vet**
•	I added an image to the hero about us section + some basic text. 
•	I changed the text on the main hero section 
•	I added styles for the two divs the about us image and the about us container. 

- [x] I am now working on the 4th section of the main page –> signup up form / button / quote.

### CODE REVIEW

**Comment by elic4vet** [comment]
+added media queries for mob + tablet version 
--> i will get back to this once i am done with the content

**Comment by elic4vet** [comment]
I added the media queries for the about us section and fixed the disportion of the image on the main about us page. Hope it is okay. I still need to check these files later too once i have some time. I tested the responsiveness with chrome dev tools. Hope it is visible

**Comment by elic4vet** [comment]
There was an issue on the mobile version of the about section concenring the main image. I fixed it but it would still get back to it later.

File: css/style.css [hunk 1]
```diff
@@ -429,6 +479,105 @@ ul {
...
+  }
+
+  .about-text {
+    font-size: 1rem;
+  }
+
+  .about-img {
+    border-radius: 20px 20px 0 0 ;
+  }
+
```
Comment (basiacarvalho) [inline css/style.css:552]: double space

File: css/style.css [hunk 2]
```diff
@@ -429,6 +479,105 @@ ul {
...
+  .about-text {
+    font-size: 1.5rem;
+  }
+  .about-img {
+    border-radius: 10px;
+  }
+
+}
+ 
+
```
Comment (basiacarvalho) [inline css/style.css:579]: empty spaces

File: index.html [hunk 3]
```diff
@@ -26,11 +26,12 @@
       <!--1st section - Hero -->
       <section class="section hero">
         <div class="container hero-container">
-          <h1 class="hero-title">This website is awesome</h1>
+          <h1 class="hero-title">Welcome to Code Studio </h1>
```
Comment (basiacarvalho) [inline index.html:29]: "CodeStudio" together as one word

File: pages/about.html [hunk 4]
```diff
@@ -24,6 +24,37 @@
       </header>
       <main>
 
+ <section class="section about-section">
+  <div class="container about-container">
+    <h1 class="about-title">About us</h1>
+    <p class="about-text">
+    Our story began in 2019, when we decided to create a website that will help people to learn how to code.
```
Comment (basiacarvalho) [inline pages/about.html:31]: structure / indentations

**Review by basiacarvalho — COMMENTED** [review]
Hi :) 
I know that you're still working on it. One important thing "CodeStudio" together as, one word. And spaces between code - but you're in the middle of coding so I  know, that it's going to change.

File: css/style.css (hunk 2 above)
Comment (elic4vet) [reply css/style.css:579]: The empty spaces is a just a way to leave some space between this section and new theme / section etc

File: css/style.css (hunk 1 above)
Comment (elic4vet) [reply css/style.css:552]: I deleted the spaces

File: index.html (hunk 3 above)
Comment (elic4vet) [reply index.html]: I changed the name

File: pages/about.html (hunk 4 above)
Comment (elic4vet) [reply pages/about.html:31]: i changed and requsted a verif.

**Review by basiacarvalho — APPROVED** [review]
At the end, when you finish with the content, remember about not leaving spaces.

**Pull request merged**